app/models/user.rb

```diff
@@ -28,7 +28,6 @@ def reply(message_decorator)
       message = message_decorator.message
       message.request = request
       message.save!
-      message.photos << message_decorator.photos
```

Comment:
I guess this is related to the other change you made :point_up_2: 
You are solving an issue that is not related to adding voice support, but you've noticed while working on `voice`... I would argue this is `code creep` :open_mouth: 
It would be nice to create a clear ticket with the issue, so that our future selves/other developers can get a quick overview of why these changes were introduced.